src/component/modal/index.tsx

```diff
@@ -112,8 +112,14 @@ function Modal(props: ModalProps) {
       }}
       nameField={{
         ...register("name", {
-          required: true,
-          maxLength: 100,
+          required: {
```

Comment:
I found that validation errors didn't appear without these being explicitly defined.